### ISSUE TITLE
generate CLI deletion of all selected resources

### ISSUE THREAD

**Issue by nelg** [iann0036/former2#9]
Hi,

As a feature request / enhancement, it would be awesome if former2 could generate AWS CLI output to delete all selected resources.    For people with Lab accounts, they often end up having AWS resources created which should be cleaned up, but with no clear way to delete everything within an AWS account. VPC deletion is OK, but misses many other types of resources.

It looks like Former2, with it's ability to scan many of the resources could maybe also output the AWS CLI syntax to just delete all the resources selected.

**Comment by iann0036** [comment]
Hi Glen,

Cool suggestion. I'm assuming you'd want this for all resource types, including those not supported by either CloudFormation and Terraform?

I'll think on this, but it does sound like a decent amount of effort regardless as I'd have to map the delete commands from every resource type so if I started it'd likely be a month or so before completion.

Thanks for raising!

**Comment by nelg** [comment]
Yes, ideally for all resource types that former2 knows how to scan, regardless of Terraform / cloudformation support.

If you implemented the feature request for importing terraform state, terraform could be used to then delete resources that it supports, by creating the terraform, importing the state and running a terraform destroy.  

However, for I think a few use cases, people just want to delete everything to clean an account, partly to reduce their bill, and partly to just get rid of mess created when running a few demos, labs, etc.  This is why I'm suggesting that if it also had the map to AWS CLI delete commands that would be pretty awesome.

One big challenge is the number of resources possible to create in AWS, so that means a lot of different API calls to delete them.  Other challenge is that some components have dependencies for deletion.. which may require people to just try again if they are not in the right order.

**Comment by PatMyron** [comment]
worth checking out https://github.com/rebuy-de/aws-nuke, pretty similar
